cladecombiner/taxon_utils.py

```diff
@@ -91,3 +97,20 @@ def printable_taxon_list(taxa: Sequence[Taxon], sep: str = "\n") -> str:
     for taxon in taxa:
         print_str += str(taxon) + sep
     return print_str
+
+
+def sort_taxa(taxa: Iterable[Taxon], taxonomy_scheme: TreelikeTaxonomyScheme):
+    assert all(isinstance(taxon, Taxon) for taxon in taxa)
+    unknown = [
```

Comment:
This functionality feels like it's repeated in a few places? It seems like this should be a method of the taxonomy scheme: `taxonomy_scheme.validate(my_taxa)`

Reply:
#15